pkgs/by-name/vu/vulkan-memory-allocator/package.nix

```diff
@@ -4,6 +4,7 @@
   fetchFromGitHub,
   fetchpatch,
   cmake,
+  gitUpdater,
```

Comment:
this is probably unnecessary. nix-update will handle git releases fine: https://github.com/GPUOpen-LibrariesAndSDKs/VulkanMemoryAllocator/releases

there's no entry here: https://nixpkgs-update-logs.nix-community.org but that might just be that the last update was in may and the package was added in June so there's been nothing to do.

Reply:
ok then